.github/workflows/perf-testing-setup.yml

```diff
@@ -47,13 +38,63 @@ jobs:
         python setup.py extension
```

Comment:
Why do we need this?

Reply:
its required for building the worker from pr.